core/sqf/src/seatrans/hbase-trx/Makefile

```diff
@@ -54,35 +65,49 @@ jdk_1_7_mapr: build_chk_mapr
 jdk_1_7_hdp: build_chk_hdp
 	$(REINSTATE_ORIG)
 	$(UNCOMMENT_STRING) HDP2.3 
-	$(MAKE) build_chk_hdp
 	set -o pipefail && $(MAVEN) -f pom.xml.hdp package -DskipTests | tee -a build_trx.log
 	mkdir -p $(MY_SQROOT)/export/lib
 	cp -pf target/$(BLD_HBASE_HDP_TRX_JARNAME) $(MY_SQROOT)/export/lib
 	$(RM) $(VFILE)
 
-jdk_1_7_apache: build_chk_apache
+jdk_1_7_apache10: build_chk_apache10
 	$(REINSTATE_ORIG)
-	$(UNCOMMENT_STRING) APACHE
-	$(MAKE) build_chk_apache
-	echo "$(MAVEN) package -DskipTests"
-	echo "### For full Maven output, see file build_trx.log"
-	set -o pipefail && $(MAVEN) -f pom.xml.apache package -DskipTests | tee build_trx.log
+	$(UNCOMMENT_STRING) HBASE1.0
+	set -o pipefail && $(MAVEN) -f pom.xml.apache10 package -DskipTests | tee build_trx.log
 	mkdir -p $(MY_SQROOT)/export/lib
-	cp -pf target/$(BLD_HBASE_APACHE_TRX_JARNAME) $(MY_SQROOT)/export/lib
+	cp -pf target/$(BLD_HBASE_APACHE10_TRX_JARNAME) $(MY_SQROOT)/export/lib
 	$(RM) $(VFILE)
+
+jdk_1_7_apache11: build_chk_apache11
 	$(REINSTATE_ORIG)
+	$(UNCOMMENT_STRING) HBASE1.1
```

Comment:
apache 11 and hbase 1.1 is used interchangeably here. Can we make this consistent so that there is no ambuiguity?
May be change the tag to Apache11?